### PR TITLE
Doc charts

### PR DESCRIPTION
**Pull request by shuchitak**
https://xmosjira.atlassian.net/browse/AP-149

In this PR I've added the doc charts generation script to run on Jenkins and archive the output plots in the artifacts.

Next step would be to add the plots to the documentation.

### CODE REVIEW

File: doc/python/README.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+The result of this script is the generation an rst file which is inclorporated into the documentation,
+incorporating a full set of charts and summary table.
+
+
+
+REQUIREMENTS
+============
+
+Python 3.9 or above. Make sure the dependencies listed in lib_src/requirements.txt are installed.
```
Comment (ed-xmos) [inline doc/python/README.txt:21]: We test with 3.10?

**Review by ed-xmos — APPROVED** [review]
Looks good - minor comment only.

**Comment by ed-xmos** [comment]
Actually one more comment - might be better to zip the image artefacts. Jenkins artefacts page is pretty cluttered.

File: doc/python/README.txt (hunk 1 above)
Comment (shuchitak) [reply doc/python/README.txt:21]: I test with Python 3.10.8 on my Mac

**Pull request merged**